repository-data/webfiles/src/main/resources/site/freemarker/common/search-strip.ftl

```diff
@@ -12,7 +12,7 @@
             <input type="text" name="query" id="query" class="search-strip__input" placeholder="${buttonLabel}" value="${query!""}" aria-label="${buttonLabel}">
         </div>
         <div class="search-strip__table-cell search-strip__table-cell--button">
-            <button class="search-strip__button" aria-label="${buttonLabel}">
+            <button id="btnSearch" class="search-strip__button" aria-label="${buttonLabel}">
```

Comment:
Assuming the `id` attribute is here so that the acceptance tests have something to target - could it be replaced with `data-uipath`?

Reply:
Good shout